src/css/application.css

```diff
@@ -71,6 +71,7 @@
 :root {
   --color-low-contrast-gray: #eee;
   --color-chrome: #e2e2e2;
+  --color-border: #bbb;
```

Comment:
_do we really need another gray tone?_

Reply:
I'd say yes. I tried the existing ones but they looked ugly and too pronounced.